### PR TITLE
Fix hda dsp dump

### PR DESCRIPTION
**Pull request by ranj063**

### CODE REVIEW

File: sound/soc/sof/intel/hda-loader.c [hunk 1]
```diff
@@ -385,11 +385,6 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
 	if (i == HDA_FW_BOOT_ATTEMPTS) {
 		dev_err(sdev->dev, "error: dsp init failed after %d attempts with err: %d\n",
 			i, ret);
-		dev_err(sdev->dev, "ROM error=0x%x: FW status=0x%x\n",
-			snd_sof_dsp_read(sdev, HDA_DSP_BAR,
-					 HDA_DSP_SRAM_REG_ROM_ERROR),
```
Comment (plbossart) [inline sound/soc/sof/intel/hda-loader.c:390]: for my education, where is HDA_DSP_SRAM_REG_ROM_ERROR read? I see a ROM_STATUS but not ROM_ERROR in the rest of the code?

Comment (ranj063) [reply sound/soc/sof/intel/hda-loader.c:390]: in hda_dsp_dump_ext_rom_status(). the ROM error would be the second dword in the extended status.

Comment (plbossart) [reply sound/soc/sof/intel/hda-loader.c:390]: ok

**Review by plbossart — APPROVED** [review]

**Review by bardliao — APPROVED** [review]

**Review by kv2019i — APPROVED** [review]
One of those things that you are like "gosh, why didn't we clean this up earlier" :) Thanks Ranjani!

**Pull request merged**